system/include/libc/setjmp.h

```diff
@@ -18,9 +18,16 @@ typedef struct __jmp_buf_tag {
 #if defined(_POSIX_SOURCE) || defined(_POSIX_C_SOURCE) \
  || defined(_XOPEN_SOURCE) || defined(_GNU_SOURCE) \
  || defined(_BSD_SOURCE)
+
+/* XXX EMSCRIPTEN: No signals support, alias sigsetjmp and siglongjmp to their non-signals counterparts. */
 typedef jmp_buf sigjmp_buf;
+#if __EMSCRIPTEN__
```

Comment:
Could you switch the lines with the `XXX comment` and `typedef jmp_buf sigjmp_buf;` around so that the changes appear "back to back", for diffs that will be easier to manage when updating musl versions.